### PR TITLE
[11.0][NEW] sale_order_secondary_unit: New module for sale in a secondary product unit

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
This module depends of https://github.com/OCA/product-attribute/pull/377

### CODE REVIEW

**Review by hieulucky111 — APPROVED** [review]

**Review by tschanzt — APPROVED** [review]
Looks good to me.

File: sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                'name': cls.product.name,
+                'product_id': cls.product.id,
+                'product_uom_qty': 1,
+                'product_uom': cls.product.uom_id.id,
+                'price_unit': 1000.00,
+            })],
+            'pricelist_id': cls.env.ref('product.list0').id,
+        })
+        so.onchange_partner_id()
+        cls.order = cls.env['sale.order'].create(
```
Comment (nikul-serpentcs) [inline sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py:46]: @sergio-teruel here use a central variable because it uses much time.

**Review by nikul-serpentcs — COMMENTED** [review]
Improve code

**Review by rousseldenis — APPROVED** [review]
Tested. It has no more interface conflicts with stock-logistics-warehouse/sale_packaging module. They can coexist.

The error is due to missing https://github.com/OCA/product-attribute/pull/377

File: sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py:46]: This is not time consuming, but a convenience for not writing long expression. Not blocking any way.

**Pull request merged**